### ISSUE TITLE
convert model to ONNX

### ISSUE THREAD

**Issue by ghost** [GHLab/fsanet-onnx-demo#1]
Hi,
Now I 'm trying to convert the model from Keras to ONNX. But I face some problem. Could you send me scripts for conversion? It is very useful to me. Thank you!

**Comment by GHLab** [comment]
@awl-ngoc-son I did get same problem from Keras so I trained it by pytorch model then converted it to onnx. There is no special code for converting pth file to onnx file.

**Comment by ghost** [comment]
Thanks for your reply. Did you convert from ONNX to MNN?

**Comment by GHLab** [comment]
@awl-ngoc-son Yes, but there was converting problem like this https://github.com/alibaba/MNN/issues/829 They will fix this issue so I'm waiting. Also NCNN converting was failed too.
